website/docs/r/ecs_service.html.markdown

```diff
@@ -93,7 +94,12 @@ For more information, see [Cluster Query Language in the Amazon EC2 Container
 Service Developer
 Guide](http://docs.aws.amazon.com/AmazonECS/latest/developerguide/cluster-query-language.html).
 
+## network_configuration
 
+`network_configuration` support the following:
+* `subnets` - (Required) The subnets associated with the task or service.
+* `security_groups` - (Optional) The security groups associated with the task or service. If you do not specify a security group, the default security group for the VPC is used.
+For more information, see [Task Networking](http://docs.aws.amazon.com/AmazonECS/latest/developerguidetask-networking.html)
```

Comment:
@atsushi-ishibashi

The link is wrong, correct url is: 

https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-networking.html

Reply:
The documentation has already been fixed in #2694

Reply:
ok, but it is still wrong, in fact, the link is https.

Reply:
@ozbillwang Sure. Thanks!